openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -780,11 +781,11 @@ The Credential Issuer indicates support for encrypted responses by including the
 
 ### Proof Types {#proof-types}
 
-This specification defines the following values for the `proof_type` property:
+This specification defines the following proof types:
 
-* `jwt`: A JWT [@!RFC7519] is used as proof of possession. When `proof_type` is `jwt`, a `proof` object MUST include a `jwt` claim containing a JWT defined in (#jwt-proof-type).
-* `cwt`: A CWT [@!RFC8392] is used as proof of possession. When `proof_type` is `cwt`, a `proof` object MUST include a `cwt` claim containing a CWT defined in (#cwt-proof-type).
-* `ldp_vp`: A W3C Verifiable Presentation object signed using the Data Integrity Proof as defined in [@VC_DATA_2.0] or [@VC_DATA], and where the proof of possession MUST be done in accordance with [@VC_Data_Integrity]. When `proof_type` is set to `ldp_vp`, the `proof` object MUST include a `ldp_vp` claim containing a [W3C Verifiable Presentation](https://www.w3.org/TR/vc-data-model-2.0/#presentations-0) defined in (#ldp-vp-proof-type).
+* `jwt`: A JWT [@!RFC7519] is used for proof of possession. When a `proof_type` parameter in a `proof` object is set to `jwt`, it MUST also contain a `jwt` parameter that includes a JWT as defined in (#jwt-proof-type). When a `proofs` object is using a `jwt` proof type, it MUST include a `jwt` parameter containing an array of JWTs as defined in (#jwt-proof-type).
```

Comment:
```suggestion
* `jwt`: A JWT [@!RFC7519] is used for proof of possession. When a `proof_type` parameter in a `proof` object is set using the member name `jwt`, its value MUST be an array of JWT, where each JWT MUST be formed as defined in (#jwt-proof-type).
```

sorry guys but this last sentece was really not clear to me.

Reply:
Your suggestion is mixing things up. The initial proposal first talks about `proof`, then about `proofs`. The latter is missing now

Reply:
I apply your ideas to the `proofs` section, if this doesn't match your expectations, please open an Issue/PR